cpp/purify/CMakeLists.txt

```diff
@@ -25,17 +25,16 @@ set(HEADERS
   "${PROJECT_BINARY_DIR}/include/purify/config.h")
 
 set(SOURCES utilities.cc pfitsio.cc
-  kernels.cc wproj_utilities.cc uvfits.cc projection_kernels.cc yaml-parser.cc )
+  kernels.cc distribute.cc wproj_utilities.cc operators.cc uvfits.cc projection_kernels.cc yaml-parser.cc )
 
 if(TARGET casacore::ms)
   list(APPEND SOURCES casacore.cc)
   list(APPEND HEADERS casacore.h)
 endif()
 
 if(PURIFY_MPI)
-  list(APPEND HEADERS mpi_utilities.h distribute.h wavelet_operator_factory.h
-                      measurement_operator_factory.h operators.h convergence_factory.h)
-  list(APPEND SOURCES mpi_utilities.cc distribute.cc operators.cc )
+  list(APPEND HEADERS mpi_utilities.h distribute.h)
+  list(APPEND SOURCES mpi_utilities.cc distribute.cc)
```

Comment:
I'm confused... you've removed 
`wavelet_operator_factory.h`, `measurement_operator_factory.h`, `operators.h` and `convergence_factory.h` however only changed `mpi_utilities.h`  and `wavelet_operator_factory.h` - did you forget to push these changes?